src/main/java/duke/commands/CreateTodo.java

```diff
@@ -0,0 +1,39 @@
+package duke.commands;
+
+import duke.ui.Ui;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+import duke.tasks.TaskList;
+import duke.storage.Storage;
+import duke.parsers.CommandParser;
+import duke.exceptions.DukeException;
+
+/**
+ * Creates a Todo Task and adds it to the TaskList.
+ */
+class CreateTodo extends Command {
+
+    public CreateTodo(CommandParser commandParser) {
+        super(commandParser);
+    }
+
+    public void execute(String arg, TaskList tasks, Ui ui, Storage storage) throws DukeException {
+        String taskName = getTaskName(arg);
+
+        Task newTask = new Todo(taskName);
+        tasks.add(newTask);
+
+        save(storage, tasks);
+
+        ui.showReply(String.format("Got it. I've added this task:\n  %s\nNow you have %d tasks in the list.", newTask,
```

Comment:
Our static analysis engine detects that your pull request includes a change that a format string `\n` instead of `%n`. According to [this description](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline), it is a bad practice, since using `\n` in format string may cause cross-platform issues.